testsuite/MDAnalysisTests/analysis/test_distances.py

```diff
@@ -186,3 +187,22 @@ def test_between_simple_case_indices_only(self):
                                                               self.ag2,
                                                               self.distance).indices)
         assert_equal(actual, self.expected)
+
+class TestImportWarnings(TestCase):
+
+    def test_no_exception_scipy_module_level(self):
```

Comment:
I think we can wrap this with the PR I put up, so it always issue a warning.  Then split the `else:` branch of this if into a separate test that only runs when the scipy module is present.